CHANGELOG.md

```diff
@@ -3,6 +3,20 @@
 > All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased](https://github.com/springload/wagtaildraftail/compare/v0.3.0...HEAD)
```

Comment:
Discussed in person – maybe we should have a formal "Unreleased" section always at the top of our changelogs.